src/App.js

```diff
@@ -1,14 +1,40 @@
 import React from "react";
-import ProPic from "./components/portfolioPic.png";
 import "./App.css";
+import NavMenu from './components/navMenu/NavMenu'
+import Home from './components/home/Home'
+import Software from './components/software/Software'
+import Music from './components/music/Music'
+import Design from './components/design/Design'
+import Footer from './components/footer/Footer'
+import { BrowserRouter, Route, Switch } from 'react-router-dom'
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={ProPic} alt="Profile Pic" />
-        <p id="Welcome">Welcome to my page!!</p>
-      </header>
+      <NavMenu />
+      <div className="Content">
+        <BrowserRouter>
+          <Switch>
+            <Route
+              path="/home"
+              component={Home}
+            />
+            <Route
+              path="/software"
+              component={Software}
+            />
+            <Route
+              path="/music"
+              component={Music}
+            />
+            <Route
+              path="/design"
+              component={Design}
+            />
+          </Switch>
+        </BrowserRouter>
+        <Footer />
+      </div>
     </div>
   );
 }
```

Comment:
Well that's simple enough... Did you put the footer in the "content" div for css and responsive deign purposes?

Reply:
The footer placement is not ideal right now. It is currently hidden until you scroll. You should try some different things until you like the way it looks.